packages/app/src/components/drawer/index.tsx

```diff
@@ -75,11 +75,12 @@ export const Drawer: React.FC<DrawerProps> = ({ showing, close, apps, confirm })
           <ScreenBody
             body={[
               <Title>
-                You{"'"}re using this account with {apps.length} other app{apps.length > 1 ? 's' : ''}.
+                You{"'"}re using this account with {apps.length} other app
```

Comment:
Can't we just write `You're` in JSX? This isn't needed, right?

Reply:
Nah - you get [an ESLint error](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md). You need to either use HTML entities (`&quot;`) or do what this is doing.